.github/workflows/release-container.yml

```diff
@@ -38,12 +45,13 @@
         uses: docker/metadata-action@v4
         with:
           images: |
-            keycloak/keycloak
-            quay.io/keycloak/keycloak
+            ${{ env.DOCKER_IMAGE }}
+            ${{ env.QUAY_IMAGE }}
           tags: |
             type=ref,event=branch
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{version}},suffix=${{ env.INCREMENTAL }}
```

Comment:
suffix may require you to add the separator (e.g. `-`) in there too? It's not well documented...

https://github.com/docker/metadata-action#typesemver doesn't really say much